<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotOriginPointLocation</class>
 <widget class="QDialog" name="RobotOriginPointLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> 原点位置</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="op_SradioButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择轴</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>绝对原点位置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="op_SlineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="op_LradioButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="op_LlineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="op_RradioButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="op_UradioButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="op_RlineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="op_UlineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="op_TradioButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="op_BradioButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="op_TlineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="op_BlineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="op_savepushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RobotOriginPointLocation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RobotOriginPointLocation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
